File: app/src/main/java/com/fernandez/pablo/la24gnc/View/DetalleAceiteFragment.java
package com.fernandez.pablo.la24gnc.View;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.fernandez.pablo.la24gnc.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class DetalleAceiteFragment extends Fragment {

    private TextView aforadorAceite1VI;
    private TextView aforadorAceite1VF;
    private TextView diferenciaAceite;
    private TextView totalLts;
    private TextView totalDineroAceite;


    public DetalleAceiteFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view = inflater.inflate(R.layout.fragment_detalle_aceite, container, false);

        this.aforadorAceite1VI = (TextView) view.findViewById(R.id.aforadorAceite1VI);
        this.aforadorAceite1VF = (TextView) view.findViewById(R.id.aforadorAceite1VF);
        this.diferenciaAceite = (TextView) view.findViewById(R.id.difAceite);

        this.totalLts = (TextView) view.findViewById(R.id.totalLts);
        this.totalDineroAceite = (TextView) view.findViewById(R.id.totalDineroAceite);

        ((DetalleTurnoActivity)this.getActivity()).cargarDetalleAceite();

        return view;
    }

    public void cargarValorInicialAceite(String valorInicial){
        this.aforadorAceite1VI.setText(valorInicial);
    }

    public void cargarValorFinalAceite(String valorFinal){
        this.aforadorAceite1VF.setText(valorFinal);
    }

    public void cargarDiferencia(String diferencia){
        this.diferenciaAceite.setText(diferencia);
    }

    public void cargarTotalLts(String totalLts){
        this.totalLts.setText(totalLts);
    }

    public void cargarTotalDinero(String totalDinero){
        this.totalDineroAceite.setText(totalDinero);
    }
}

File: app/src/main/java/com/fernandez/pablo/la24gnc/View/DetalleTurno/DetalleAceiteFragment.kt
package com.fernandez.pablo.la24gnc.View.DetalleTurno


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView

import com.fernandez.pablo.la24gnc.R


/**
 * A simple [Fragment] subclass.
 */
class DetalleAceiteFragment : Fragment() {

    private var aforadorAceite1VI: TextView? = null
    private var aforadorAceite1VF: TextView? = null
    private var diferenciaAceite: TextView? = null
    private var totalLts: TextView? = null
    private var totalDineroAceite: TextView? = null


    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        val view = inflater!!.inflate(R.layout.fragment_detalle_aceite, container, false)

        this.aforadorAceite1VI = view.findViewById(R.id.aforadorAceite1VI) as TextView
        this.aforadorAceite1VF = view.findViewById(R.id.aforadorAceite1VF) as TextView
        this.diferenciaAceite = view.findViewById(R.id.difAceite) as TextView

        this.totalLts = view.findViewById(R.id.totalLts) as TextView
        this.totalDineroAceite = view.findViewById(R.id.totalDineroAceite) as TextView

        (this.activity as DetalleTurnoActivity).cargarDetalleAceite()

        return view
    }

    fun cargarValorInicialAceite(valorInicial: String) {
        this.aforadorAceite1VI!!.text = valorInicial
    }

    fun cargarValorFinalAceite(valorFinal: String) {
        this.aforadorAceite1VF!!.text = valorFinal
    }

    fun cargarDiferencia(diferencia: String) {
        this.diferenciaAceite!!.text = diferencia
    }

    fun cargarTotalLts(totalLts: String) {
        this.totalLts!!.text = totalLts
    }

    fun cargarTotalDinero(totalDinero: String) {
        this.totalDineroAceite!!.text = totalDinero
    }
}// Required empty public constructor
